sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TriggerExecutor.scala

```diff
@@ -43,7 +42,9 @@ case class OneTimeExecutor() extends TriggerExecutor {
 /**
  * A trigger executor that runs a batch every `intervalMs` milliseconds.
  */
-case class ProcessingTimeExecutor(processingTime: ProcessingTime, clock: Clock = new SystemClock())
+case class ProcessingTimeExecutor(
+    processingTime: ProcessingTimeTrigger,
```

Comment:
Please rename the variable together.
- `processingTime` -> `processingTimeTrigger`.
- `private val intervalMs = processingTime.intervalMs` -> `private val intervalMs = processingTimeTrigger.intervalMs`